packages/framework/src/hooks/useHtmlPassThrough.tsx

```diff
@@ -23,14 +27,6 @@ export const useHtmlPassThrough = (
             }
           });
         }
-
-        if (testId) {
-          try {
-            (node as HTMLElement).setAttribute("data-testid", testId);
-          } catch (e) {
-            error(e);
-          }
-        }
```

Comment:
Why does this fix the issue? It's the same call as what's in the for loop.

Reply:
@evshi honestly I have no idea that why this fix resolved by removing this, I just goes through the code structure and noticed that we no need to use this, as we can manage this with the for loop

Reply:
Firefox shows the error more descriptively.
I believe this is the issue with the way we are using rootRef in antd Table. Getting the table containerzied inside a div with ref={rootRef} solves the problem without disturbing useHtmlPassThrough file

<img width="1800" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/32386713/8a73e4f6-44e5-483c-8d83-5c14cce44ddf">

Reply:
Interesting - I wonder why putting it into a single map works. This probably means the ref changes...I think wrapping it in a div is reasonable.

Reply:
@anserwaseem as per my view, no need to remove the changes from useHtmlPassThrough hook, as I just combine the 2 different attributes into one loop.